src/widgets/configure/configure.ts

```diff
@@ -11,12 +11,12 @@ export type ConfigureWidgetOptions = PlainSearchParameters;
 
 export type ConfigureWidget = WidgetFactory<{}, ConfigureWidgetOptions>;
 
-const configure: ConfigureWidget = widgetParams => {
+const configure: ConfigureWidget = function configure(widgetOptions) {
   // This is a renderless widget that falls back to the connector's
   // noop render and unmount functions.
   const makeWidget = connectConfigure(noop);
 
-  return makeWidget({ searchParameters: widgetParams });
+  return makeWidget({ searchParameters: widgetOptions });
```

Comment:
Is this because of ConfigureWidget _Options_ ?

Reply:
Actually, we have `widgetParams` in some widgets, and `widgetOptions` in others. I am just to unify the style.

Reply:
So is [this](https://github.com/algolia/instantsearch.js/blob/chore/typescript-migration-configures/src/types/widget.ts#L202:L202) going to be changed to `widgetOptions` in the end? (meaning the public signature changes along with the docs)

Reply:
Nice catch. It's the other way around. It should be widgetParams. Going to update the pull request.

Reply:
Done, updating the named `widgetParams` vs `widgetOptions` in other pull request.

Reply:
Just confirmed 👍